workers/unity/Assets/Fps/Scripts/SetupLogic/OneTimeInitialisation.cs

```diff
@@ -23,6 +23,7 @@ private static void Init()
 
             // Setup template to use for player on connecting client
             PlayerLifecycleConfig.CreatePlayerEntityTemplate = FpsEntityTemplates.Player;
+            PlayerLifecycleConfig.AutoRequestPlayerCreation = false;
```

Comment:
just a note, you shouldn't have to set this to false since you do `PlayerLifecycleHelper.AddClientSystems(Worker.World, false);`